src/main/webapp/WEB-INF/view/chat.jsp

```diff
@@ -56,6 +59,15 @@ List<Message> messages = (List<Message>) request.getAttribute("messages");
       var chatDiv = document.getElementById('chat');
       chatDiv.scrollTop = chatDiv.scrollHeight;
     };
+	function playMsg(m) {
+	  console.log('inside func');
```

Comment:
You probably don't need these kinds of print statements. They're useful for debugging, but probably not for the real product. It's not a big deal though so it's up to you!